react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: ''
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(id) {
+    if (this.state.id === id) {
+      this.setState({ id: '' });
+    } else {
+      this.setState({ id: id });
+    }
+  }
+
+  render() {
+    const books = this.props.books;
+    const bookItems = books.map(books =>
+      <div className='author' onClick={() => this.handleClick(books.id)} key={books.id}>{books.author}
+        <div className={`content ${books.id === this.state.id ? 'title show' : 'title hidden'}`}>{books.title}</div>
+      </div>);
+    return (
+      <div>{bookItems}</div>
+    );
```

Comment:
You don't need the extra div here and should avoid extra divs when possible. You can just do:

```JSX
return bookItems;
```

Or you can even just directly return the result of `map`:

```JSX
return books.map(...);
```